gigascience/gigadb-website
# a11y - team page

# Pull request for issue: #1369

This is a pull request for the following functionalities:

* Fix accessibility issues in the team page http://gigadb.gigasciencejournal.com:9170/site/team
* Improve layout appearance

## How to test?

* Navigate here http://gigadb.gigasciencejournal.com:9170/site/team and run a11y audit (WAVE chrome extension)
* Check and verify the new layout

## How have functionalities been implemented?

* Added new less module for team grid
* Restructured entire team page to use a new layout
  * Looks visually better (better card alignment since now each card fills the same space as its siblings in the same row
  * Collapsible text for long descriptions allows to maintain layout from looking too imbalanced
  * Better semantic html by replacing table as layout by a CSS grid with articles as items

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
